.github/workflows/run-tests.yml

```diff
@@ -64,3 +59,16 @@ jobs:
   test_backoffice:
     name: Test Backoffice
     uses: ./.github/workflows/test-backoffice.yml
+
+  test_service:
```

Comment:
@asvegah Ah, this might not be correct. Looks like it's working against a real environment? Will investigate options for testing the service before merging.

Reply:
Turns out the environment is not used on those checks. I've removed it.